scripts/compare.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+
+import sys
+import getopt
+import shapely.geometry
+from svg.path import parse_path as parse_svg_path
+from svg.path import Path as SvgPath
+from svg.path import Move as SvgMove
+from svg.path import Line as SvgLine
+from svg.path import Close as SvgClose
+from svg.path import CubicBezier as SvgCubicBezier
+from svg.path import Arc as SvgArc
+from xml.dom import minidom
+
+stroke_width = 1.25
+
+#FIXME:
+width="2222.5mm"
+height="2470.1499mm"
+viewBox="0 0 2222.5 2470.1499"
```

Comment:
Stolen from the existing file; but does not work because of #9.
Even if we were to fix the viewBox or applied group transforms, then we'd still need transforms for some path elements.